chromadb/test/property/test_embeddings.py

```diff
@@ -372,23 +391,30 @@ def wait_for_compaction(self) -> None:
 
     @rule(
         target=embedding_ids,
-        record_set=strategies.recordsets(collection_st),
+        record_set=strategies.recordsets(collection_st, can_ids_be_empty=True),
     )
     def add_embeddings(self, record_set: strategies.RecordSet) -> MultipleResults[ID]:
         res = super().add_embeddings(record_set)
         normalized_record_set: strategies.NormalizedRecordSet = invariants.wrap_all(
             record_set
         )
+
```

Comment:
Looking at this logic again, I'm confused about this change. Previously everything was from the length of the record_set ...